nifi-external/nifi-kafka-connect/nifi-kafka-connector-tests/pom.xml

```diff
@@ -61,10 +61,22 @@
             <version>2.0.0-SNAPSHOT</version>
             <type>nar</type>
         </dependency>
+        <dependency>
+            <groupId>commons-io</groupId>
+            <artifactId>commons-io</artifactId>
+            <version>2.13.0</version>
```

Comment:
This version number should be removed because it is set in the root Maven configuration.